src/Flashcards.test.js

```diff
@@ -30,4 +31,21 @@ describe('<Flashcards />', () => {
       />);
     expect(wrapper.find(Flashcard).length).toEqual(1);
   });
+
+  it('renders the index of the current flashcard', () => {
+    const wrapper = shallow(
+      <div className='Flashcards-index'>
+        <Chip>
+          1 / 6
+        </Chip>
+      </div>
```

Comment:
What are we testing here? We should be testing our `Flashcards` component, not toolbox's `Chip` component.